dockers/skylab/HCA_post_processing/Dockerfile

```diff
@@ -8,6 +8,7 @@ COPY requirements.txt .
 RUN pip3 install numpy==1.17.0
 RUN pip3 install cython==0.29.15
 RUN pip3 install -r requirements.txt
+RUN pip3 install -U  pytest
```

Comment:
I don't think we need to add the test (and especially the tests data) to the dockerfile. We can think about how to run these in an automated way at a later time, but I imagine we will do something like the sctools tests (which set up an environment and then run pytest). A this is here we would only test the code that is built into the docker, which is not necessarily useful for PR tests

Reply:
removed.